### PR TITLE
New params

### PR DESCRIPTION
**Pull request by vondele**
current set of params that leads to the best net from scratch so far

https://tests.stockfishchess.org/tests/view/5ff40fc86019e097de3eee8f
(run 79).

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -97,10 +97,13 @@ def forward(self, us, them, w_in, b_in):
   def step_(self, batch, batch_idx, loss_type):
     us, them, white, black, outcome, score = batch
 
-    q = self(us, them, white, black)
-    t = outcome
-    # Divide score by 600.0 to match the expected NNUE scaling factor
-    p = (score / 600.0).sigmoid()
+    # 600.0 to match the expected NNUE scaling factor
+    nnue2score = 600
```
Comment (Sopel97) [inline model.py:102]: could we perhaps rename this, or at least document, that this is `kPonanzaConstant` from the nodchip trainer?

Comment (vondele) [reply model.py:102]: like:
```# 600.0 is the kPonanzaConstant scaling factor needed to convert the training net output to a score```
?

Comment (Sopel97) [reply model.py:102]: that and that it needs to match in stockfish

Comment (vondele) [reply model.py:102]: in stockfish, or in the serializer ?

Comment (Sopel97) [reply model.py:102]: Right, sorry, in the serializer. Stockfish doesn't need it obviously because it's baked into the weights.

**Comment by Sopel97** [comment]
For completeness:
Master:
https://tests.stockfishchess.org/tests/view/5fee3c006019e097de3eea8d
https://tests.stockfishchess.org/tests/view/5feeebde6019e097de3eead0
https://tests.stockfishchess.org/tests/view/5fef930f6019e097de3eeb84
https://tests.stockfishchess.org/tests/view/5ff0b6f76019e097de3eec52

PR:
https://tests.stockfishchess.org/tests/view/5ff40fc86019e097de3eee8f
https://tests.stockfishchess.org/tests/view/5ff4ef836019e097de3eef3d

**Pull request merged**